news/12434.doc.rst

```diff
@@ -0,0 +1 @@
+Replace some links to PEPs with links to the canonical specifications on https://packaging.python.org
```

Comment:
How about
```suggestion
Replaced some links to PEPs with links to the canonical specifications on :py:doc:`pypug:index`
```

Reply:
You're right about using Intersphinx.

I'd keep "Replace", though, since the other news entries are written in the present tense.

Reply:
```suggestion
Replace some links to PEPs with links to the canonical specifications on the :doc:`pypug:index`
```